src/pydocstyle/violations.py

```diff
@@ -235,5 +236,6 @@ def __getattr__(self, item):
     'pep257': all_errors - {'D203', 'D212', 'D213', 'D214', 'D215', 'D404',
                             'D405', 'D406', 'D407', 'D408', 'D409', 'D410',
                             'D411'},
-    'numpy': all_errors - {'D203', 'D212', 'D213', 'D402'}
+    'numpy': all_errors - {'D203', 'D212', 'D213', 'D402'},
+    'pep8': all_errors - {'D216'}
```

Comment:
Can you add a trailing `,` here? It will reduce future visual noise created by folks adding other style names.